### PR TITLE
Chart

### PR DESCRIPTION
**Pull request by priolette**
Add option to choose year displayed

### CODE REVIEW

File: Energy Saver/Pages/Statistics.cshtml [hunk 1]
```diff
@@ -6,11 +6,54 @@
...
 <h1 class="text-color">Statistics</h1>
-<div><canvas id="lineChart"></canvas></div>
+<div class="container-fluid">
+    <div class="row">
+        <div class="col-lg-2">
+            <form asp-page-handler="year" method="post">
+                <div class="form-group">
+                    @Html.AntiForgeryToken()
+                    <select class="form-control" name="Year" id="yearDropdown" onchange="editYear()">
+                        @for (int i = DateTime.Now.Year; i > 1919; i--)
```
Comment (Leminode) [inline Energy Saver/Pages/Statistics.cshtml:16]: you could make year choices dynamic, depending on whether that year has an entry

File: Energy Saver/Pages/Statistics.cshtml [hunk 2]
```diff
@@ -6,11 +6,54 @@
...
+                        {
+                            <option value="@i">@i</option>
```
Comment (Tauras-Rim) [inline Energy Saver/Pages/Statistics.cshtml:18]: We could add a custom filter for showing data. Not necessarily a yearly analysis

**Review by Leminode — APPROVED** [review]
Good changes overalll

**Review by Tauras-Rim — APPROVED** [review]
Good job, the chart looks way better now

**Pull request merged**